tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -299,7 +299,7 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 		// Wait for PV and PVC to Bind
-		framework.ExpectNoError(fpv.WaitOnPVandPVC(client, namespace, pv, pvc))
+		framework.ExpectNoError(fpv.WaitOnPVandPVC(client, framework.NewTimeoutContextWithDefaults(), namespace, pv, pvc))
```

Comment:
what is the value for context time out here?

Reply:
For PV-PVC bind, it's 3 minutes - https://github.com/kubernetes/kubernetes/blob/v1.21.1/test/e2e/framework/timeouts.go#L31